metadatastore/mds.py

```diff
@@ -78,6 +78,8 @@ def _runstart_col(self):
 
             self.__runstart_col.create_index([('uid', pymongo.DESCENDING)],
                                              unique=True)
+            self.__runstart_col.create_index([('time', pymongo.DESCENDING)],
```

Comment:
We do not need this index, we have other index that starts with time https://docs.mongodb.org/manual/core/index-compound/#prefixes